src/main/java/ganymedes01/etfuturum/world/nether/dimension/NetherChunkProvider.java

```diff
@@ -189,7 +189,7 @@ public void generateNetherTerrain(int par1, int par2, Block[] blocks, byte[] aby
 	 * name based on ChunkProviderGenerate
 	 */
 	public void replaceBlocksForBiome(int x, int z, Block[] blocks, byte[] metas, BiomeGenBase[] par4ArrayOfBiomeGenBase) {
-		ChunkProviderEvent.ReplaceBiomeBlocks event = new ChunkProviderEvent.ReplaceBiomeBlocks(this, x, z, blocks, par4ArrayOfBiomeGenBase);
```

Comment:
Keep it this way; If a mod extends the world height your new revision may not be compatible (if I'm wrong let me know)